apps/web/src/app/[locale]/(dashboard)/[wsId]/inventory/products/edit/form.tsx

```
@@ -0,0 +1,461 @@
+                            )}
+                          >
+                            <SelectValue
+                              placeholder={t(
+                                'ws-inventory-categories.placeholder'
+                              )}
+                            />
+                          </SelectTrigger>
+                        </FormControl>
+                        <SelectContent>
+                          {categories.map((category) => (
+                            <SelectItem value={category.id}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)